### DISCUSSION TITLE
Graph view

### DISCUSSION THREAD

**Discussion by MitPitt** [outline/outline#1692] [Ideas]
Since Outline detects backward links to internal documents, a cool feature would be a graph view of the whole knowledge database. I think Obsidian.md implemented it best, check them out.

And please don't forget about the document tags feature, it is highly needed.

**Comment by tommoor** [comment]
Hi hi, interested how do you use this feature in Obsidian?

**Reply by MitPitt** [reply]
This is extremely useful if you use Obsidian as a personal knowledge manager. Read about Zettelkasten method. Basically you make a bunch of atomic notes that mention each other and/or have repeating tags. And when you look at the graph, it's easier to start noticing emerging connections between stuff you previously couldn't see, and that gives you new ideas and so on.

Many see graph view as one of the strongest selling points of Obsidian.

Outline can be a knowledge manager too. And not just for personal use. I want to use this feature to make a studying platform for my community. This would greatly help in enabling knowledge exchange between people studying different disciplines, because there is always a lot of interdisciplinary stuff to be learnt. Like sociology can use maths, design can use computer graphics knowledge and so on.

A graph view is not easy to implement, I presume, so it's kind of an end-game feature.
What is actually really needed right now is document tagging (for all the same reasons and more), I'm surprised Outline doesn't have that yet.

**Reply by RickCogley** [reply]
Such a view is great a few years in, when you’re trying to tidy up, or find all the places a doc links to (esp if you want to replace that doc with a new version).

**Reply by gemtokenfactory** [reply]
https://www.nuclino.com/ has it. we need it for outline or notion

**Reply by almereyda** [reply]
Graph views are becoming a general feature in "webbed notebooks" as those used for "digital gardens" that happen to grow these days. Please find inspiration in how this is solved in:

* [logseq/logseq](https://github.com/logseq/logseq)
* [zadam/trilium](https://github.com/zadam/trilium)
* [athensresearch/athens](https://github.com/athensresearch/athens)

and covered more broadly in

* [Added graph visualization of notes by JulienMir · Pull Request #921 · Zettlr/Zettlr](https://github.com/Zettlr/Zettlr/pull/921#)

**Reply by almereyda** [reply]
The Zettlr PR eventually also concluded with [an implementation of a graph view](https://docs.zettlr.com/en/academic/graph/) of the link topology within the document repository.

**Comment by MadalinNitu** [comment]
Is this still in pending? Is a cool idea.

**Comment by sudonoodle** [comment]
+1. Something like Obsidian Canvas (https://obsidian.md/canvas) would be awesome!

**Comment by Alekuso** [comment]
+1 it is incredibly useful on docs that mentions other docs
